spec/fortnox/api/validators/context.rb

```diff
@@ -13,7 +13,7 @@
   end
   shared_examples_for 'invalid' do |attribute, values, error_type|
     values.each do |value|
-      context "when #{attribute} set to #{value}" do
+      context "when :#{attribute} set to #{value}" do
```

Comment:
This is fine, if we are sure they will always be symbols. If you wanted something more generic you could do `"#{ attribute.inspect }"` instead so that it prints the value with the right representation regardless of class.